include/vcpkg/base/expected.h

```diff
@@ -134,13 +134,13 @@ namespace vcpkg
         // Constructors are intentionally implicit
 
         ExpectedT(const S& s, ExpectedRightTag = {}) : m_s(s) { }
-        template<class = std::enable_if<!std::is_reference<S>::value>>
+        template<std::enable_if_t<!std::is_reference<S>::value, int> = 0>
```

Comment:
is there a reason you are not using:
```suggestion
        template<std::enable_if_t<!std::is_reference_v<S>, int> = 0>
```